components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -51,9 +46,19 @@
           <div class="filter-label">Unknown</div>
           <div class="filter-total">{{ serviceGroupsHealthSummary.unknown }}</div>
         </chef-option>
+        <chef-option class="filter connected" value='connected' (click)="updateHealthFilter('connected')" (keyup.enter)="updateHealthFilter('connected')">
+          <chef-icon class="filter-icon">link</chef-icon>
+          <div class="filter-label">Connected</div>
+          <div class="filter-total">{{ serviceGroupsHealthSummary.total - serviceGroupsHealthSummary.disconnected }}</div>
```

Comment:
the connected count isn't in the API yet, however its just the total minus the disconnected, so we can calculate this on the frontend for the time being.